rules/windows/execution_apt_solarwinds_backdoor_child_cmd_powershell.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/12/14"
+maturity = "production"
+updated_date = "2020/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious SolarWinds child process (Cmd.exe or Powershell.exe) was detected. Verify process details such as network
+connections and file writes.
```

Comment:
_Verify process ..._ may fit better in `false_positives`

Reply:
done